service/worker/service.go

```diff
@@ -197,6 +197,7 @@ func (s *Service) Start() {
 	logger.Info("worker starting", tag.ComponentWorker)
 
 	s.Resource.Start()
+	s.Resource.GetDomainReplicationQueue().Start()
```

Comment:
Is worker service the only service need DomainReplicationQueue? If so then probably we should consider move domain replication queue out of resource.

Reply:
Both frontend and worker will use it